spec/interactor/declaration_spec.rb

```diff
@@ -125,6 +125,22 @@ def context
         c.foo = 'bar'
         expect(c.foo).to eq('bar')
       end
+
+      context 'with default value' do
+        let(:declared) {
+          build_declared do
+            hold foo: 'bar'
```

Comment:
you could also add a test with a Proc

Reply:
added a spec checking that we generate a different instance thanks to proc